### PR TITLE
upgrade fixes and minor changes in uuid documentation

### PR DESCRIPTION
**Pull request by bradymiller**
Still testing

@stephenwaite , added another "clearing" of the insurance_data date field because of the time gap in the database changes and codebase changes.

### CODE REVIEW

File: sql/5_0_2-to-5_0_3_upgrade.sql [hunk 1]
```diff
@@ -590,6 +594,11 @@ ALTER TABLE `api_res_ver`
     ADD CONSTRAINT `fk_resver_forcedid` FOREIGN KEY (`forced_id_pid`) REFERENCES `api_forced_id` (`pid`);
 #EndIf
```
Comment (bradymiller) [inline sql/5_0_2-to-5_0_3_upgrade.sql:596]: will add this comment here:
```
-- Note the below line will also be skipped if the uuid_registry table does not yet exist
```

**Comment by dixonwhitmire** [comment]
Hi @bradymiller I'm adding in UUID conversion functions (in my neverending branch) to the UuidRegistry. I'm plugging these into the patient validator and service I rolled for the FHIR implementations, and so far it's looking good.

I am also adding test cases for these new functions to a UuidRegistryTest class. Once the class is in the code base it would be helpful to add coverage for the other methods in UuidRegistry. If there are existing test cases for UuidRegistry and I missed them in a volley of merges, just toss me a link.

Thanks!
```
    /**
     * Converts a UUID byte value to a string representation
     * @return the UUID string value
     */
    public static function uuidToString($uuidBytes)
    {
        return Uuid::fromBytes($uuidBytes)->toString();
    }

    /**
     * Converts a UUID string to a bytes representation
     * @return the UUID bytes value
     */
    public static function uuidToBytes($uuidString)
    {
        return Uuid::fromString($uuidString)->getBytes();
    }
```

**Comment by bradymiller** [comment]
@dixonwhitmire , Great idea on the conversion functions.

**Comment by bradymiller** [comment]
testing well and bringing this into codebase

**Pull request merged**